flow/templates/stampede2.sh

```diff
@@ -55,33 +54,11 @@ export LAUNCHER_JOB_FILE={{ launcher_file }}
 $LAUNCHER_DIR/paramrun
 rm {{ launcher_file }}
 {% else %}
-{% set cmd_suffix = cmd_suffix|default('') ~ (' &' if parallel else '') %}
-{% for operation in operations %}
-
-# {{ "%s"|format(operation) }}
-{{ "_FLOW_STAMPEDE_OFFSET_=%d "|format(operation.directives['nranks']|return_and_increment) }}{{ operation.cmd }}{{ cmd_suffix }}
-{% if operation.eligible_operations|length > 0 %}
-# Eligible to run:
-{% for run_op in operation.eligible_operations %}
-# {{ run_op.cmd }}
-{% endfor %}
-{% endif %}
-{% if operation.operations_with_unmet_preconditions|length > 0 %}
-# Operations with unmet preconditions:
-{% for run_op in operation.operations_with_unmet_preconditions %}
-# {{ run_op.cmd }}
-{% endfor %}
-{% endif %}
-{% if operation.operations_with_met_postconditions|length > 0 %}
-# Operations with all postconditions met:
-{% for run_op in operation.operations_with_met_postconditions %}
-# {{ run_op.cmd }}
-{% endfor %}
-{% endif %}
-{% endfor %}
+{% block pre_operation %}
```

Comment:
```suggestion
{# Only the pre_operation block is overridden, all other behavior is inherited from base_script.sh #}
{% block pre_operation %}
```

Reply:
But I'm not even sure this will work -- the code you're replacing has the variable defined on the same line as the operation command.

Before:
```bash
_FLOW_STAMPEDE_OFFSET_=8 ibrun ... stuff
```

After:
```bash
_FLOW_STAMPEDE_OFFSET_=8
ibrun ... stuff
```

@vyasr can you confirm this and offer insight on how we should proceed, since you suggested the change to Stampede2? Do we add an export like `export _FLOW_STAMPEDE_OFFSET_=8` or do we just abandon this idea entirely and copy-paste the whole template as before?

Reply:
@bdice you are correct that the variable scope will not work with the intervening newline, and that adding an export should fix it. That is probably the best solution since it also has the benefit of significantly reducing the duplication here.